cdap-explore/src/main/java/co/cask/cdap/explore/guice/ExploreRuntimeModule.java

```diff
@@ -307,6 +310,18 @@ public boolean accept(String className, URL classUrl, URL classPathUrl) {
     }
     List<String> orderedDependenciesStr = builder.build();
 
+    // These dependency files need to be copied over to spark container
+    System.setProperty(BaseHiveExploreService.SPARK_YARN_DIST_FILES,
```

Comment:
Yes, it is in Explore YARN container. This logic needs to be cleaned up a bit later.